changelog.md

```diff
@@ -1,3 +1,10 @@
+### UNRELEASED
```

Comment:
Since we are not takking anythin extra, i think bump to v4.3 at the same time, maybe just in a sperate commit, so proting would be easier

Reply:
Fixed